e2e/rbd.go

```diff
@@ -465,6 +465,21 @@ var _ = Describe("RBD", func() {
 					e2elog.Failf("failed to create PVC and application with error %v", err)
 				}
 
+				appOpt := metav1.ListOptions{
+					LabelSelector: fmt.Sprintf("app=%s", app.Name),
+				}
+				// TODO: Remove this once we ensure that rbd-nbd can sync data
+				// from Filesystem layer to backend rbd image as part of its
+				// detach or SIGTERM signal handler
+				_, stdErr, err := execCommandInPod(
+					f,
+					fmt.Sprintf("cd %s && sync", app.Spec.Containers[0].VolumeMounts[0].MountPath),
```

Comment:
Not sure if changing the directory makes a difference. Consider running `sync <mountpath>` instead?

Reply:
Sure, that looks like a good idea.